webapp/src/components/WaveformChart.tsx

```
@@ -70,6 +130,30 @@ export function WaveformChart({ waveforms, frequency }: WaveformChartProps) {
           </CardTitle>
+          <div className="flex gap-1" data-testid="period-toggle">
+            <button
+              onClick={() => setNumPeriods(1)}
+              className={`px-2 py-0.5 text-xs rounded border ${
+                numPeriods === 1
+                  ? "bg-blue-600 border-blue-500 text-white"
+                  : "border-gray-600 text-gray-400 hover:text-gray-200"
+              }`}
+              data-testid="btn-1t"
+            >
+              1T
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)